src/components/input.rs

```diff
@@ -1,16 +1,15 @@
 //! Input Validation Components.
 
-use bitcoin::{Address, Amount, FeeRate, Transaction, Txid, consensus};
 use dioxus::prelude::*;
 
 #[cfg(debug_assertions)]
 use dioxus::logger::tracing::trace;
-use secp256k1::schnorr;
 
 use crate::{
     ESPLORA_ENDPOINT, NETWORK,
     esplora::FeeEstimate,
-    util::{npub_to_address, parse_network, parse_npub, parse_nsec},
+    util::{npub_to_address, parse_network, parse_npub},
+    validation::*,
```

Comment:
no globbing import the stuff explicitly, this is not Python...
We only glob preludes like `dioxus::prelude::*` or in tests with `use super::*`